packages/elements-dev-portal/src/version.ts

```diff
@@ -1,2 +1,2 @@
 // auto-updated during build
-export const appVersion = '1.0.11';
+export const appVersion = '1.1.0';
```

Comment:
... lol. Looks right though!
![image](https://user-images.githubusercontent.com/587740/129776608-4af8b108-b281-4031-9591-3f3e0ccb5135.png)

Reply:
Oh goodness this snuck in and I don't know exactly how. Should I revert?

Reply:
No, lets leave it. It probably should have happened as part of an earlier commit. The comment above makes me think it's some automated build step.